db/sources/chartcons-jokes.html

```diff
@@ -0,0 +1,1149 @@
+				 	     	 	     	 <div class="wp-content entry"><p>Laughter is like a medicine to the soul. Most times when one is facing challenges, it becomes had to laugh, but good jokes does the magic of making you laugh even in the mist of dare circumstances. Now how do you know a good joke? even when there are lots of jokes out there. Most good jokes does not necessarily make sense, but you will certainly still find it funny. One of this kind of joke is the laffy taffy joke. Lets take a look.</p>
+<h2><span style="text-decoration: underline;">Best Laffy Taffy Jokes -Funny Laffy Taffy Jokes</span></h2>
+<p>1. Why did the strawberry cross the road? Because his mother was in a jam.</p>
+<p>2. How do you keep a lion from charging? Take away its credit cards.</p>
+<p>3. What do you call a cow with a twitch? Beef jerky</p>
+<p>4. What is the best way to keep water from running? Don’t pay the water bill</p>
+<p>5. Where do cows go to have fun? The moo-vies!</p>
+<p>6. What time was is when the elephant sat on a chair? Time to get a new chair.</p>
+<p>7. What did the flower say to the bike? Petal</p>
+<p>8. Why do we not tell secrets in a corn patch? Too many ears!</p>
+<p>9. What do you call a crab who plays baseball? — a pinch-hitter</p>
+<p>10. What is the clumsiest bee? — a bumbling bee</p>
+<p>11. What kind of bean can’t grow? — a jelly bean</p>
+<p>12. Whats green and fluffy and comes from mars? — a martian mellow</p>
+<p>13. How does a man on the moon get his hair cut? — eclipse it</p>
+<p>14. What time is it when an elephant sits on your fence? — time to get a new fence.</p>
+<p>15. What is green, red, and runs 100 mph? — a frog in a blender</p>
+<p>16. How do you turn soup into gold? — add 24 carrots (karats)</p>
+<p>17. What do you do if a rhino charges you? — Give him your credit card.</p>
+<p>18. Why did they bury the battery? — Because it was dead.</p>
+<p>19. What do sneezes wear on their feet? — ahh-shoes</p>
+<p>20. What do wolves say when they are introduced? — howl do you do?</p>
+<p>21. What does a car run on? — wheels</p>
+<p>22. Who greets you at a haunted house? — a host ghost</p>
+<p>23. Why did the farmer bury all his money? — to make his soil rich</p>
+<p>24. Where can you find an ocean without water? — on a map</p>
+<p>25. What do you call a horse that likes arts &amp; crafts? — a hobby horse</p>
+<p>26. Why do shoemakers go to heaven? Because they have good soles</p>
+<p>27. What do you call an avid gardener? — herb</p>
+<p>28. What happens when you cross a singer and a rocking chair? — you rock to the beat.</p>
+<p>29. How do you mend a broken jack o’ lantern? — with a pumpkin patch.</p>
+<p>30. what do you call a lease of false teeth? — a dental rental</p>
+<p>31. Where did the kittens go on the class trip — to the meow-seum.</p>
```

Comment:
Looks like this one missed a `?` End up with an odd parsing. Maybe add it here in the source file?